WebRoot/css/zm.css

```diff
@@ -1637,6 +1669,9 @@ INPUT[type="text"].search_input-expanded:focus {
 	overflow:auto;
 }
 
+.attBubbleHolder > span:not(:first-child) {
+	@AttachmentChipSpacing@
```

Comment:
In our code, have seen many places where we do styling based on child position and proceeded with the same.